requirements.txt

```diff
@@ -41,3 +41,4 @@ pyvo==1.1
 lxml==4.6.3
 suds-py3==1.4.3
 vcrpy==4.1.1
+aiosmtpd==1.4.2
```

Comment:
Since our only need is to complete an SMTP transaction, would https://docs.python.org/3/library/smtpd.html not suffice?

Reply:
I started with smtpd but then was led to this package instead when attempting to stand up a server supporting the `starttls` method. My impression was that smtpd was slightly outdated and didn't really support TLS servers out-of-the-box.

Reply:
Ah, ok, and we need TLS specifically?

We should avoid new dependencies when possible, but if it's useful and simplifies things dramatically it's ok.

Reply:
The TLS functionality is used in our STMP client code before sending emails ([here](https://github.com/skyportal/skyportal/blob/41a5e099056b5bc6a9cb3976ed7bb9aeec0c72b9/skyportal/email_utils.py#L26)). I think the alternative is to add something in the configs to say whether to use TLS or not when sending emails over SMTP, which honestly wouldn't be much work either. It just felt weird to add a conditional to the existing API logic to accommodate the test server instead of having the other way around.